Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Image/StreamingImageAssetCreator.h

```diff
@@ -48,7 +48,9 @@ namespace AZ
 
             //! Set the average color of the image.
             void SetAverageColor(Color avgColor);
-            
+
+            void AddTag(AZ::Name tag);
```

Comment:
would it be more efficient to pass by const reference?
```suggestion
            void AddTag(const AZ::Name& tag);
```